app/controllers/users_controller.rb

```diff
@@ -125,6 +125,22 @@ def profile
                     .joins(:node)
                     .limit(20)
     @wikis = wikis.collect(&:parent).uniq
+    @twitter = nil 
+    @facebook = nil
+    @github = nil 
+    if User.user_has_power_tag("twitter" , @user.uid) == true 
```

Comment:
Hi, this is looking great! One suggestion is that you could use `@user.has_power_tag` here, since `@user` exists -- and save a little repeated code in `user.rb`. Then we wouldn't have to worry about testing any new code there either! Also `@user.power_tag('tagname')` should return the value. Thanks!

Reply:
Thank you @jywarren sir ! 
I was not able to find has_power_tag('tagname') and power_tag('tagname') in User.rb .

Reply:
Oh i found those functions in Node.rb model...but i am not sure we can call those using @user instance . What do you think ? Thank you !